lib/cisco_node_utils/cisco_cmn_utils.rb

```diff
@@ -367,5 +367,27 @@ def self.add_quotes(value)
         value.start_with?('"') && value.end_with?('"')
       value
     end # add_quotes
+
+    # extract value of property from aggregate-address
```

Comment:
This is a general purpose method right?  Should you use a more generic comment that does not mention aggregate-address?  

Thanks for moving these into `cisco_cmd_utils` so they can be used by any objects that need them.